client_test.go

```diff
@@ -38,6 +38,53 @@ func TestGetMarathonURL(t *testing.T) {
 	assert.Equal(t, endpoint.Client.GetMarathonURL(), endpoint.URL)
 }
 
+func TestApiRequest(t *testing.T) {
```

Comment:
`s/Api/API/` to satisfy [Go's convention on initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).